src/outputs/intram_by_itl.py

```diff
@@ -102,8 +102,7 @@ def output_intram_by_itl(
 
     """
     # Declare Config Values
-    NETWORK_OR_HDFS = config["global"]["network_or_hdfs"]
-    OUTPUT_PATH = config[f"{NETWORK_OR_HDFS}_paths"]["output_path"]
+    OUTPUT_PATH = config["outputs_paths"]["outputs_master"]
```

Comment:
Not to do with the code, but there is a a folder in outputs for intram_by_itl1: can that be taken out now?  Also the schema still has a 1 in it, and the detailed mapper name: small thing, but should they be updated as they cover itl 1 and 2 now?

Reply:
good point, I've removed it.